tests/test_snapshots.py

```diff
@@ -18,7 +18,7 @@ def test_unicode_string(snapshot):
 def test_multiple_snapshots(snapshot):
     assert "First." == snapshot
     snapshot.assert_match("Second.")
-    assert "Third." == snapshot
+    snapshot("Third.")
```

Comment:
Oh we should probably remove this syntax

Reply:
I agree, but any particular reasons why?